Marlin/src/lcd/language/language_en.h

```diff
@@ -128,6 +128,9 @@ namespace Language_en {
 
     LSTR MSG_PREHEAT_2                    = _UxGT("Preheat ") PREHEAT_2_LABEL;
     LSTR MSG_PREHEAT_2_SETTINGS           = _UxGT("Preheat ") PREHEAT_2_LABEL _UxGT(" Conf");
+    #ifndef PREHEAT_3_LABEL
+      #define PREHEAT_3_LABEL "Custom"
```

Comment:
Wouldn’t it make more sense to require/sanity check a third preheat option being set for your Pro UI instead of this one-off label being set in the language file?

Reply:
I agree.

Reply:
I tried to remove the PREHEAT_3_LABEL definition out from the language file and define in the configuration file and I getting this error in the test script: https://github.com/MarlinFirmware/Marlin/runs/6168076291?check_suite_focus=true#step:7:41

Reply:
It will be even more convenient once E3V2 has the equivalent to `lcd_put_u8str_ind_P` so it can do substitutions in menu items and other labels. Then we can eliminate many translated strings.
```cpp
/**
 * lcd_put_u8str_ind_P
 *
 * Print a string with an index substituted within it:
 *
 *   $ displays the clipped C-string given by the inStr argument
 *   = displays  '0'....'10' for indexes 0 - 10
 *   ~ displays  '1'....'11' for indexes 0 - 10
 *   * displays 'E1'...'E11' for indexes 0 - 10 (By default. Uses LCD_FIRST_TOOL)
 *   @ displays an axis name such as XYZUVW, or E for an extruder
 */
```